### ISSUE TITLE
Add game manual support

### ISSUE THREAD

**Issue by mmatyas** [mmatyas/pegasus-frontend#447]
Needs cross-platform PDF rendering and some QML-side interface and API.

**Comment by bleuge** [comment]
This could be great, imagine a pdf browser, that we could add pre and post execution commands, i could pre- send it to a translation program, or whatever i want to do previous loading it, well maybe the post is not necessary (at least I cannot find a nice example)

**Comment by mmatyas** [comment]
Yup, I was also thinking about mupdf (which seems to work everywhere) or poppler (which in theory integrates well with Qt). Will experiment with both of them.

I feel the PDF events to be overkill though; the data and assets for the games are expected to be in the format you'd like to see on the screen, so there are no events for the rest of the localizale text, eg. descriptions.

**Comment by sargant** [comment]
Resurrecting this old issue as I'd love to see this functionality. For ease of implementation, would it be sufficient to just open PDFs in the default file handler for the OS?

**Comment by mmatyas** [comment]
I'm afraid no, as on some platforms there can't be more than one program accessing the screen. Desktop PDF readers also rarely support gamepad controls, and of course there's the chance that the user just don't have a PDF viewer.
